lib/mountain_view.rb

```diff
@@ -11,3 +10,5 @@ def self.configure
     yield(configuration)
   end
 end
+
+require 'mountain_view/engine' if defined?(Rails)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.